2-environments/README.md

```diff
@@ -37,3 +37,22 @@ The purpose of this step is to set up dev, nonprod, and prod environments within
     1. Run `terraform init`
     1. Run `terraform plan` and review output
     1. Run `terraform apply`
+
+### Run tf-wrapper.sh script locally
+1. Change into 2-environments folder.
+1. Run `cp ../build/tf-wrapper.sh .`
+1. Run `chmod 755 ./tf-wrapper.sh`
+1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Update backend.tf with your bucket from bootstrap. You can run
+```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```
+1. Run `./tf-wrapper.sh init dev`
```

Comment:
Add a line or two before these explaining why we do `./tf-wrapper.sh apply branch`. Perhaps something like `We will now deploy each of our environments(dev/prod/stage) using this script. When checked into a VCS, each  environment corresponds to a branch and only the corresponding environment is applied.`

Reply:
Yeah! Adding your suggestion!